### PR TITLE
Create tests for institution model and for RemoveAdminPermissionsInInstitutionHierarchy handler

### PR DESCRIPTION
**Pull request by ruanres**
**Feature/Bug description:** There were no tests for them both.

**Solution:** The tests were created.

**TODO/FIXME:** n/a

### CODE REVIEW

File: backend/test/model_test/institution_test.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        )
+        
+
+    def test_unfollow(self):
+        # case in which the user is not a member
+        self.institution.follow(self.user.key)
+        self.assertEquals(self.institution.followers, [self.user.key],
+            "Institution followers should have user key"
+        )
+        self.institution.unfollow(self.user.key)
```
Comment (raonismaneoto) [inline backend/test/model_test/institution_test.py]: Shouldn't you retrieve the institution before asserting?

File: backend/test/model_test/institution_test.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        # verifies inst admin also have permissions to child_a
+        self.assertEquals(actual_permissions, expected_permissions,
+            "The admin does not have the expected permissions"
+        )
+
+        # Case 3: Get all permission and the admin has one institution and 
+        # one child that he is not admin  
+        # Institution(admin) -> child_a(other_admin) -> child_b(admin) -> child_c(other_admin) -> x
+        child_b = generate_child_to_parent(child_a, self.admin)
+        import pdb
```
Comment (raonismaneoto) [inline backend/test/model_test/institution_test.py]: remove it

File: backend/test/model_test/institution_test.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        expected_permissions = generate_permissions(child_a.key.urlsafe(), expected_permissions)
+        expected_permissions = generate_permissions(child_b.key.urlsafe(), expected_permissions)
+        expected_permissions = generate_permissions(child_c.key.urlsafe(), expected_permissions)
+        actual_permissions = self.institution.get_hierarchy_admin_permissions()
+        # verifies institution admin also have permissions to child_a
+        self.assertEquals(actual_permissions, expected_permissions,
+            "The admin does not have the expected permissions"
+        )    
+
+        # pdb.set_trace()
```
Comment (raonismaneoto) [inline backend/test/model_test/institution_test.py]: remove it

File: backend/test/model_test/institution_test.py (hunk 1 above)
Comment (ruanres) [reply backend/test/model_test/institution_test.py]: I tried retrieving the institution and not retrieving it before the assertion, and the test passed anyway.

**Pull request merged**